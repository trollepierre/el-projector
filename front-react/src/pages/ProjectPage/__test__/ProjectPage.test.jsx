import React from 'react'
import { shallow } from 'enzyme'
import { ProjectPage } from '../../index'

describe('ProjectPage', () => {
  it('should match snapshot', () => {
    // When
    const wrapper = shallow(<ProjectPage/>)

    // Then
    expect(wrapper).toMatchSnapshot()
  })
})
