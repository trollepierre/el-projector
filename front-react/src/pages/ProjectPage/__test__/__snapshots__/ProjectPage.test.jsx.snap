// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectPage should match snapshot 1`] = `
<div
  className="page"
>
  <ProjectForm />
  <ProjectTable />
</div>
`;
